x/gov/codec.proto

```diff
@@ -133,6 +133,13 @@ message Proposal {
   bytes raw_option = 3;
 }
 
+// Resolution contains TextResolution and an electorate reference.
+message Resolution {
+  weave.Metadata metadata = 1;
+  orm.VersionedIDRef electorate_ref = 2 [(gogoproto.nullable) = false];
```

Comment:
I think I also requested the original proposal_id to be included here

Reply:
I'm using the original ticket description as reference as it was not clear what takes precedence. 
If it does not reflect the requirements - may I ask you to update it with latest?

Reply:
Yep, true. This is still early stages WIP and I committed what I had to fix another PR.